workbench-core/accounts/src/services/costCenterService.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { GetItemCommandOutput } from '@aws-sdk/client-dynamodb';
+import { AwsService, resourceTypeToKey, buildPkSk, uuidWithLowercasePrefix } from '@aws/workbench-core-base';
+import Boom from '@hapi/boom';
+import Account from '../models/account';
+import CostCenter from '../models/costCenter';
+import CreateCostCenter from '../models/createCostCenter';
+import AccountService from './accountService';
+
+export default class CostCenterService {
+  private _aws: AwsService;
+  private readonly _tableName: string;
+
+  public constructor(constants: { TABLE_NAME: string }) {
+    const { TABLE_NAME } = constants;
+    this._tableName = TABLE_NAME;
+    this._aws = new AwsService({ region: process.env.AWS_REGION!, ddbTableName: TABLE_NAME });
+  }
+
+  public async getCostCenter(costCenterId: string): Promise<CostCenter> {
+    // Get by id
+    const response = (await this._aws.helpers.ddb
+      .get(buildPkSk(costCenterId, resourceTypeToKey.costCenter))
+      .execute()) as GetItemCommandOutput;
+
+    if (response.Item === undefined) {
+      throw Boom.notFound(`Could not find cost center ${costCenterId}`);
+    }
+    const costCenter = response.Item as unknown as CostCenter;
+
+    return Promise.resolve(costCenter);
+  }
+
+  public async create(createCostCenter: CreateCostCenter): Promise<CostCenter> {
```

Comment:
Having a model here means that we can reuse these params as needed whenever we need to build the object that is used to create a cost center. Its a pattern we should be using when interfaces are used in more than one place (such as in costCenterService.ts and costCenterService.test.ts). It also has the added benefit of reducing code duplication because the objects properties are gathered together. If you make a change in the service, the test automatically gets those changes for free, reducing the chance of update or copy/past errors.

Reply:
I'm ok with this new approach. We'll need to backfill this approach for `EnvironmentService` and `AccountService` though. 

I'm thinking in the future, we can move `schemas` to `swb-app`. This will allow the routing package to determine it's own request params. However, that request would then be mapped to a `Create<ResourceEntity>Request` before it can be passed to the `<ResourceEntity>Service`.